react/components/AddressList.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { AddressContext } from 'vtex.address-context'
+import { PlaceDetails } from 'vtex.place-components'
+import { Address } from 'vtex.checkout-graphql'
+import { OrderShipping } from 'vtex.order-shipping'
+import { FormattedMessage } from 'react-intl'
+import { ListGroup, GroupOption } from 'vtex.checkout-components'
+
+interface Props {
+  addresses: Address[]
+  onCreateAddress?: () => void
+  onAddressSelected?: (address: Address) => void
+  selectedAddress: Address | null
+}
+
+const AddressList: React.FC<Props> = ({
+  addresses,
+  onCreateAddress = () => {},
+  onAddressSelected = () => {},
+  selectedAddress,
+}) => {
+  const { countries } = OrderShipping.useOrderShipping()
+
+  return (
+    <div>
+      <span className="db fw6 lh-copy mb3">
```

Comment:
Why not use a `div` if the element is of type `block` via `db`?

Reply:
because a `span` is better semantically than a div to contain text

Reply:
Good point.